oauthproxy_test.go

```diff
@@ -986,3 +986,47 @@ func TestAjaxForbiddendRequest(t *testing.T) {
 	mime := rh.Get("Content-Type")
 	assert.NotEqual(t, applicationJSON, mime)
 }
+
+func TestClearSplittedCookie(t *testing.T) {
+	p := OAuthProxy{CookieName: "oauth2"}
+	var rw = httptest.NewRecorder()
+	req := httptest.NewRequest("get", "/", nil)
+
+	req.AddCookie(&http.Cookie{
+		Name:  "test1",
+		Value: "test1",
+	})
+	req.AddCookie(&http.Cookie{
+		Name:  "oauth2-0",
+		Value: "oauth2-0",
+	})
+	req.AddCookie(&http.Cookie{
+		Name:  "oauth2-1",
+		Value: "oauth2-1",
+	})
+
+	p.ClearSessionCookie(rw, req)
+	header := rw.Header()
+
+	assert.Equal(t, 3, len(header["Set-Cookie"]), "should have 3 set-cookie header entries")
+}
+
+func TestClearNotSplittedCookie(t *testing.T) {
```

Comment:
Nit: Could we rename this to `TestClearSingleCookie`?

Reply:
Yep